packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -1689,6 +1705,200 @@ describe("wrangler", () => {
 			});
 		});
 	});
+	describe("local", () => {
+		it("should modify local kv storage", async () => {
+			await runWrangler(
+				`kv:key get val --namespace-id some-namespace-id --local --text`
+			);
+			expect(std.out).toMatchInlineSnapshot(`""`);
+
+			await runWrangler(
+				`kv:key put val value --namespace-id some-namespace-id --local`
+			);
+			expect(std.out).toMatchInlineSnapshot(`
+			"
+			Writing the value \\"value\\" to key \\"val\\" on namespace some-namespace-id."
+		`);
+
+			await runWrangler(
+				`kv:key get val --namespace-id some-namespace-id --local --text`
+			);
+			expect(std.out).toMatchInlineSnapshot(`
+			"
+			Writing the value \\"value\\" to key \\"val\\" on namespace some-namespace-id.
+			value"
```

Comment:
Is there a way to clear std.out between commands, that would make these tests a lot cleaner

Reply:
You could make the current `it` a `describe`, and then use a separate `it` for each of these commands?

Reply:
Yes I split up the test into a few different tests